addons/website_forum/static/src/js/website_forum.share.js

```diff
@@ -52,11 +52,15 @@ var ForumShare = publicWidget.registry.socialShare.extend({
         } else if (this.targetType === 'social-alert') {
             $question.before(qweb.render('website.social_alert', {medias: this.socialList}));
         } else {
-            $('body').append(qweb.render('website.social_modal', {
+            this.el.insertAdjacentHTML('beforeend', qweb.render('website.social_modal', {
                 medias: this.socialList,
                 target_type: this.targetType,
                 state: $question.data('state'),
             }));
+            const modalEl = this.el.querySelector('#oe_social_share_modal');
+            modalEl.addEventListener('hidden.bs.modal', () => {
+                modalEl.remove();
+            });
```

Comment:
If I am not mistaken, this diff does not solve a bug. If so, I would suggest to remove it in order to do at least modification as possible on the DOM (as we are in stable) :+1: